core/tests/test_asset_incident_report_model.py

```diff
@@ -23,3 +28,48 @@ def test_model_string_representation(self):
             str(incident_report),
             f"{incident_report.incident_type}: {incident_report.asset}",
         )
+
+
+class TestAssetIncidentReportNotifications(CoreBaseTestCase):
+    """
+    Test Notifications elicited by Asset Incident Report creation
+    """
+
+    # Asset_model = apps.get_model("core", "Asset")
+    User_model = apps.get_model("core", "User")
+    Notifications_model = apps.get_model("core", "Notifications")
+
+    def setUp(self):
+        self.admin_user = self.User_model.objects.create(
+            email="newuser@andela.com",
+            password="devpassword",
+            first_name="first",
+            last_name="second",
+            phone_number="254720900900",
+            location=self.centre,
+            department=self.department,
+            is_securityuser=True,
+            is_superuser=True,
+        )
+
+    def test_admins_are_notified_when_notification_is_created(self):
+        original_notifications_count = self.Notifications_model.objects.count()
+        admins = self.User_model.objects.filter(is_superuser=True)
+        admin_count = admins.count()
+        report = AssetIncidentReport.objects.create(
+            asset=self.test_asset, incident_type=constants.LOSS, submitted_by=self.user
+        )
+        notifications = self.Notifications_model.objects.filter(target__in=admins)
+        notification = notifications[0]
+        incident_url = "{}/{}".format(reverse("incidence-reports-list"), report.id)
+        body = "{} submitted a new incident report {}.".format(
```

Comment:
Let's stick to [fstrings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) for formatting. @kenneth051 what's your take on this?